No way to delete link with webapp

#### How Shlink is set-up

* Shlink Version: 3.0.2
* PHP Version: 8.0
* How do you serve Shlink: Self-hosted Apache
* Database engine used: MySQL

#### Summary
During installation I set "Visits threshold" to 30. If I try to delete a link in webapp (version 3.5.1) with 1 visit I read this message: "Impossible to delete short URL with short code "xxxxx", since it has more than "1" visits."

No problem if I use cli command to delete it

#### Current behavior

No link is deleted

#### Expected behavior

Link is deleted

#### How to reproduce

create a link and try to delete it in webapp

Shlink Version: 3.0.2
Shlink Web App 3.51
PHP Version: 8.1
How do you serve Shlink: Self-hosted Nginx
Database engine used: MariaDB

I have the exact same issue as above and receive the same error message from within the web app. 

However, unlike above, I also cannot delete the affected shorturls from cli. When attempting to delete via cli, I receive a message saying "[ERROR] No URL found with short code "xxxx" for domain "https://xxx/xxxx". However, the short code (same as the confirmation code entered to confirm deletion within the web app) is correct. 

I tried the fix just posted on two different instances of shlink on different servers -- changed the value in config/params/generated_config.php to 'DELETE_SHORT_URL_THRESHOLD' => 100, cleared cache, restarted Nginx, php, tried using different browsers, etc., and the issues still persist on both servers. 

Not sure if there is anything else I should do to have the changes take effect?

> very strange... I edit this value in generated_config.php but without effect. I restart apache too... 

@saiballo I forgot to tell you also have to delete the `data/cache/app_config.php` file. Then it will start working.

I have updated my original comment in case I need to reference to it.

> [ERROR] No URL found with short code "xxxx" for domain "https://xxx/xxxx"

@dfnsar715 This is an unrelated issue. If you have applied the mitigation described here, you will only be able to delete short URLs not in the default domain, if you explicitly pass the domain with `--domain example.com`.

The shlink-web-client does this automatically, that's why the error is different.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Unreleased]
+### Added
+* *Nothing*
+
+### Changed
+* [#1382](https://github.com/shlinkio/shlink/issues/1382) Updated docker image to PHP 8.1.3.
+
+### Deprecated
+* *Nothing*
+
+### Removed
+* *Nothing*
+
+### Fixed
+* [#1377](https://github.com/shlinkio/shlink/issues/1377) Fixed installer always setting delete threshold with value 1.
+
+
 ## [3.0.2] - 2022-02-10
 ### Added
 * *Nothing*

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM php:8.1.1-alpine3.15 as base
+FROM php:8.1.3-alpine3.15 as base
 
 ARG SHLINK_VERSION=latest
 ENV SHLINK_VERSION ${SHLINK_VERSION}

--- a/composer.json
+++ b/composer.json
@@ -51,7 +51,7 @@
         "shlinkio/shlink-config": "^1.6",
         "shlinkio/shlink-event-dispatcher": "^2.3",
         "shlinkio/shlink-importer": "^2.5",
-        "shlinkio/shlink-installer": "^7.0.1",
+        "shlinkio/shlink-installer": "^7.0.2",
         "shlinkio/shlink-ip-geolocation": "^2.2",
         "symfony/console": "^6.0",
         "symfony/filesystem": "^6.0",

--- a/data/infra/php.Dockerfile
+++ b/data/infra/php.Dockerfile
@@ -1,4 +1,4 @@
-FROM php:8.1.1-fpm-alpine3.15
+FROM php:8.1.3-fpm-alpine3.15
 MAINTAINER Alejandro Celaya <alejandro@alejandrocelaya.com>
 
 ENV APCU_VERSION 5.1.21

--- a/data/infra/swoole.Dockerfile
+++ b/data/infra/swoole.Dockerfile
@@ -1,4 +1,4 @@
-FROM php:8.1.1-alpine3.15
+FROM php:8.1.3-alpine3.15
 MAINTAINER Alejandro Celaya <alejandro@alejandrocelaya.com>
 
 ENV APCU_VERSION 5.1.21

